docs/hosting/community-edition-features.md

```diff
@@ -0,0 +1,30 @@
+---
+#https://www.notion.so/n8n/Frontmatter-432c2b8dff1f43d4b1c8d20075510fe4
+description: Community edition features
```

Comment:
```suggestion
title: Community edition features
description: Differences in available features between the Community edition and other paid plans.
```

Reply:
maybe something like this? @Johannesishere

Reply:
Yes. Thank you.